.github/workflows/parse-branch-name/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Parse Branch Name
+description: Parse the branch name, and set parts as environment variable.
```

Comment:
Add explanation of `CLEAN_PACKAGE_NAME`, `PACKAGE_VERSION`, `CHANNEL`
Also in the action itself so the reader would not have to look in the composite

Reply:
Done